docs/installation/setup.rst

```diff
@@ -102,7 +136,11 @@ For more detail(s), refer to the official Red Hat documentation:
     - `Configure repositories with YUM`_
     - `Advanced repositories configuration`_
 
-Example OpenStack deployment
-----------------------------
+ETCD
```

Comment:
Not an acronyme, so should be `etcd` I think

Reply:
right